RecoPixelVertexing/PixelVertexFinding/plugins/gpuVertexFinderImpl.h

```diff
@@ -112,37 +118,42 @@ namespace gpuVertexFinder {
     init(soa, ws_d.get());
     loadTracks(tksoa, soa, ws_d.get(), ptMin);
 #endif
-
+    
 #ifdef __CUDACC__
+    // Running too many thread lead to problems when printf is enabled.
+    constexpr int maxThreadsForPrint = 1024 - 256;
+    constexpr int splitBlocks = 1024;
+    constexpr int splitThreadsPerBlock = 128;
```

Comment:
could you use the names
```suggestion
    constexpr int numBlocks = 1024;
    constexpr int threadsPerBlock = 128;
```
?

Reply:
Sure, fixed in latest commits.